### PR TITLE
Make the bootstrap notification alert fixed-top.

### PR DESCRIPTION
**Pull request by vdbergh**
In that way it will always be visible.

### CODE REVIEW

**Comment by vdbergh** [comment]
As @dav1312 pointed out this does not work in dark mode as alerts are semi-transparent. I have not figured out yet what bootstrap incantations are necessary to fix this so making this PR draft for now.

**Comment by vdbergh** [comment]
Fixed I think.

**Comment by vdbergh** [comment]
> I'm currently testing on DEV this https://getbootstrap.com/docs/5.2/utilities/opacity/

I think this does not work because opacity is cumulative. But let's see.

**Comment by ppigazzini** [comment]
Opacity didn't worked, updating DEV with your last commit.

**Review by ppigazzini — APPROVED** [review]

File: server/fishtest/static/css/application.css [hunk 1]
```diff
@@ -612,3 +612,7 @@ input[type="checkbox"],
   pointer-events: none;
   opacity: 0.5;
 }
+
+/* remove transparency of background */
+.alert-success-non-transparent {
+}
```
Comment (dav1312) [inline server/fishtest/static/css/application.css:618]: Empty rule?

**Comment by ppigazzini** [comment]
PROD update, sorry @dav1312 :(

**Comment by dav1312** [comment]
It's fine, it doesn't change anything so

File: server/fishtest/static/css/application.css (hunk 1 above)
Comment (ppigazzini) [reply server/fishtest/static/css/application.css:618]: If I understood correctly we can drop this rule. Let me know that I will push directly a commit.

Comment (ppigazzini) [reply server/fishtest/static/css/application.css:618]: @dav1312 BTW fell free to open a PR to clean up the rule yourself :)

**Comment by vdbergh** [comment]
@dav1312  I wasn't sure if it is legal to refer to a class that is not defined.

**Comment by ppigazzini** [comment]
BTW I'm testing what happens with multiple bootstrap notifications.

**Comment by vdbergh** [comment]
Currently they will overwrite each other. I will make a PR where they will append instead.

**Comment by vdbergh** [comment]
It seems it is legal to refer to a class that has no associated style. That would mean the empty rule can go.

**Pull request merged**